DeepSpeech.ipynb

```diff
@@ -1387,7 +1389,8 @@
     "            avg_tower_gradients = average_gradients(tower_results[1])\n",
     "\n",
     "            # Add logging of averaged gradients\n",
-    "            log_grads_and_vars(avg_tower_gradients)\n",
+    "            if log_variables:\n",
+    "                log_grads_and_vars(avg_tower_gradients)\n",
```

Comment:
Are we sure that this is not touching the model layout? It would be enough to just exclude the merged summaries retrieval from `session.run`. However: If this is not the case, I am fine with it.